CustomCollections/CustomCollections/ContentView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ContentView.swift
+//  CustomCollections
+//
+//  Created by Полуянов Яков on 01.10.2023.
+//
+
+import SwiftUI
+
+struct ContentView: View {
```

Comment:
Should have created an UIKit empty project but you've created SwiftUI project instead. This works though we can change it later